test/correctness/simd_op_check.cpp

```diff
@@ -1755,8 +1769,14 @@ struct Test {
         check("vnot(v*)", hvx_width/2, ~u16_1);
         check("vnot(v*)", hvx_width/4, ~u32_1);
 
-        check("vsplat(r*)", hvx_width/1, in_u8(0));
-        check("vsplat(r*)", hvx_width/2, in_u16(0));
+        if (is_v62) {
+            // v62 - Broadcasting unsigned 8 bit and 16 bit scalers
+            // check("v*.b = vsplat(r*)", hvx_width/1, in_u8(0));
+            // check("v*.h = vsplat(r*)", hvx_width/2, in_u16(0));
```

Comment:
This is the vsplat problem I was facing. Actually this instruction should ideally be generated, but presently the instruction doesn't actually generate even though there is a call to use the intrinsic that generates this instruction. What should I do about this for the time being?